src/tests/topp/OpenSwathWorkflow_22_output.chrom.mzML

```diff
@@ -425,11 +425,11 @@
 	<index name="chromatogram">
 		<offset idRef="PEPTIDEA_Precursor_i0">3268</offset>
 		<offset idRef="PEPTIDEA_Precursor_i0">5564</offset>
-		<offset idRef="PEPTIDEB_Precursor_i0">7860</offset>
-		<offset idRef="PEPTIDEB_Precursor_i0">10156</offset>
-		<offset idRef="tr1">12452</offset>
-		<offset idRef="tr2">14767</offset>
-		<offset idRef="3">17082</offset>
+		<offset idRef="tr1">7860</offset>
+		<offset idRef="tr2">10175</offset>
+		<offset idRef="3">12490</offset>
+		<offset idRef="PEPTIDEB_Precursor_i0">14803</offset>
+		<offset idRef="PEPTIDEB_Precursor_i0">17099</offset>
 		<offset idRef="4">19395</offset>
 		<offset idRef="5">21708</offset>
 		<offset idRef="6">24021</offset>
```

Comment:
@cbielow the output is the same its just a re-ordering of the MS1 chromatograms. They should be the same because the offsets map to exactly the same byte positions in the file, e.g. the chromatogram 4 is at byte 19395 before and after